content/post/web-analytics-process.md

```diff
@@ -0,0 +1,90 @@
+---
+author: Ana Kravitz
+title: "Web Analytics Maintenance Process"
+publishDate: "2018-07-28"
```

Comment:
@anakravitz we've got you scheduled for "2018-07-08" right now. If you're good with that you can change the publishdate and drop the `draft` tag :)